### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by jsdabrowski**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,8 @@
 ## Purpose of this project
 
 
-The repository is used for learning puposes.. Do not treat it seriously as we just play with it. Good work. Very good. Very good.
+The repository is for learning puposes. Do not treat it seriously as we just play with it. Good work. Very good. Very good.
+
+Will be using the file for a short period
```
Comment (jackdabrowski) [inline README.md]: Add period.

File: README.md [hunk 2]
```diff
@@ -3,4 +3,8 @@
...
+
+Thank
```
Comment (jackdabrowski) [inline README.md]: Add period.

File: README.md [hunk 3]
```diff
@@ -3,4 +3,8 @@
 ## Purpose of this project
 
 
-The repository is used for learning puposes.. Do not treat it seriously as we just play with it. Good work. Very good. Very good.
```
Comment (jackdabrowski) [inline README.md:6]: This seems to be good.

**Review by jackdabrowski — CHANGES_REQUESTED** [review]

**Comment by jsdabrowski** [comment]
All comments resolved, please take another look.

**Review by jackdabrowski — APPROVED** [review]

**Pull request merged**